packages/react/src/components/Dashboard/DashboardGrid.mdx

```diff
@@ -0,0 +1,273 @@
+- [Source Code](#source-code)
+- [Feedback](#feedback)
+
+## Getting started
+
+The simplest way to use the dashboard grid is pass it a set of cards and let it figure out its own layout to use.
+
+```jsx
+import { DashboardGrid, CARD_SIZES, CARD_TYPES, Card } from 'carbon-addons-iot-react';
+
+<DashboardGrid>
+  {[
+    <Card
+      title="Facility Metrics"
+      key="facility"
+      id="facility"
+      size={CARD_SIZES.MEDIUM}
+      type={CARD_TYPES.VALUE}
+      availableActions={{
+        delete: true,
+      }}
+      content="My Facility Metrics"
+    />,
+  ]}
+</DashboardGrid>;
+```
+
+## Custom layout
+
+Pass a custom layout to the dashboard grid to assign positions and sizes to the various to the cards.
+In this case, only the lg and md breakpoint have a custom layout defined. Resize the screen to see
+the cards reposition and resize themselves at different layouts. In this example, when the screen matches
+the lg breakpoint size, the card with the id 'facility' will be in the fifth row and fifth column
+(zero-based index), the card with the id 'humidity' will be in the first column and first row, and the
+card with the id 'utilization' will be in the fourth column and first row. However, since the humidity
+card already occupied that space, it will get pushed down to the next row by the auto-reflowing of the
+grid so that no cards overlap.
+
+```jsx
+<DashboardGrid
+  layouts={{
+    lg: [
+      { i: 'facility', x: 4, y: 4, w: 1, h: 1 },
+      { i: 'humidity', x: 0, y: 0, w: 1, h: 1 },
+      { i: 'utilization', x: 3, y: 0, w: 1, h: 1 },
+    ],
+    md: [
+      { i: 'facility', x: 0, y: 0, w: 1, h: 1 },
+      { i: 'humidity', x: 0, y: 1, w: 1, h: 1 },
+      { i: 'utilization', x: 0, y: 2, w: 1, h: 1 },
+    ],
+  }}
+>
+  {[
+    <Card
+      title="Facility Metrics"
+      key="facility"
+      id="facility"
+      size={CARD_SIZES.MEDIUM}
+      type={CARD_TYPES.VALUE}
+      availableActions={{
+        delete: true,
+      }}
+      content="My Facility Metrics"
+    />,
+    <Card
+      title="Humidity"
+      key="humidity"
+      id="humidity"
+      size={CARD_SIZES.SMALL}
+      type={CARD_TYPES.VALUE}
+      availableActions={{
+        delete: true,
+      }}
+      content="My Humidity Values"
+    />,
+    <Card
+      title="Utilization"
+      id="utilization"
+      key="utilization"
+      size={CARD_SIZES.SMALL}
+      type={CARD_TYPES.VALUE}
+      availableActions={{
+        delete: true,
+      }}
+      content="My utilization chart"
+    />,
+  ]}
+</DashboardGrid>
+```
+
+## Editable layout
+
+You can drag and drop the cards around. The `onLayoutChange` handler will be triggered each time you
+drop a card in a new location. The card array contains objects representing each card in the layout.
+Similar to the layout prop above. It's easiest to inspect the output in the Actions tab of the
+[is editable story](/?path=/story/1-watson-iot-dashboard-grid--dashboard-is-editable) to see the
```

Comment:
Link is not working, remove the section `/?path=` 
Story links must start with `/story/`